content/docs/_index.md

```diff
@@ -9,15 +9,18 @@ aliases:
 
 Bank-Vaults provides the following tools for Hashicorp Vault to make its use easier and more automated:
 
-- A [Kubernetes operator for provisioning secrets]({{< relref "/docs/operator/_index.md" >}}).
-- A [mutating webhook for injecting secrets]({{< relref "/docs/mutating-webhook/_index.md" >}}).
-- A [CLI tool]({{< relref "/docs/cli-tool/_index.md" >}}) to automatically initialize, unseal, and configure Vault with authentication methods and secret engines.
-- A [Go client wrapper]({{< relref "/docs/go-library/_index.md" >}}) for the official Vault client with automatic token renewal, built-in Kubernetes support, and a dynamic database credential provider.
+- [Bank-Vaults CLI tool]({{< relref "/docs/cli-tool/_index.md" >}}) makes working with Hashicorp Vault easier. For example, it can automatically initialize, unseal, and configure Vault.
```

Comment:
```suggestion
- [Bank-Vaults CLI]({{< relref "/docs/cli-tool/_index.md" >}}) makes working with Hashicorp Vault easier. For example, it can automatically initialize, unseal, and configure Vault.
```